Refactor BuyerInformation

BeedleEiEi opened the pull request:
Use lombok to reduce line of code

Commit:
Refactor BuyerInformation

Use lombok to reduce line of code

champillon reviewed (commented):
แก้ test มาด้วย

BeedleEiEi commented:
Test เดิมที่ใช้ ไม่ fail ครับ เพราะเป็นการ refactor ให้จำนวนบรรทัดลดลง

champillon commented on src/main/java/th/ac/kmitl/it/soa/group7/models/BuyerInformation.java:
```diff
@@ -1,68 +1,13 @@
+import lombok.Data;
+
+public @Data class BuyerInformation {
 	public String taxId;
```
ตัวแปรพวกนี้ต้องเป็น public เหรอ ควรเป็น private รึป่าว?

champillon reviewed (commented):
ฝากชี้แจงที

Commit:
Change access modifier from public to private

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into Refactor-BuyerInformation

BeedleEiEi commented:
แก้ให้แแล้วครับ @champillon

Commit:
Change access modifier from private to public

//Because of jackson Json builder can't be private it's require public
for auto build json

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into Refactor-BuyerInformation

champillon commented:
0.5 คะแนน
